### DISCUSSION TITLE
How can I install Steam for this web OS?

### DISCUSSION THREAD

**Discussion by theLucidLibrarian** [firecracker-microvm/firecracker#4227] [Q&A]
Ok, so I recently found this webOS and since ChromeOS sucks, I thought I could install Steam on here, but I have no idea how. I found this link that runs the OS, but I barely know how it works.
https://anura.mercurywork.shop/
Is this even possible? Thanks in advance!

**Comment by wearyzen** [comment]
Hi @ususausausa,
Please have a look at our https://github.com/firecracker-microvm/firecracker/blob/main/CODE_OF_CONDUCT.md and refrain from using inappropriate language towards any project.

Also, I think the request is out of scope for Firecracker. Please have a look at https://github.com/firecracker-microvm/firecracker/blob/main/README.md for more on what Firecracker offers.

**Reply by theLucidLibrarian** [reply]
Ah, gotcha!
